software/apps/audio_module/event_detector/Makefile

```diff
@@ -0,0 +1,17 @@
+# makefile for user application
+
+# the current directory
+APP_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+# files needed for this code
+C_SRCS   := $(wildcard *.c) 
```

Comment:
Hi Pat,
Thanks for the follow-up. And no worries at all, I understand the burden of a paper deadline. We'll get this working soon.
In the meantime, godspeed to your paper!